clients/go/admin/config.go

```diff
@@ -40,6 +40,7 @@ type Config struct {
 	Endpoint              config.URL      `json:"endpoint" pflag:",For admin types, specify where the uri of the service is located."`
 	UseInsecureConnection bool            `json:"insecure" pflag:",Use insecure connection."`
 	InsecureSkipVerify    bool            `json:"insecureSkipVerify" pflag:",InsecureSkipVerify controls whether a client verifies the server's certificate chain and host name. Caution : shouldn't be use for production usecases'"`
+	CAcerts               string          `json:"caCerts" pflag:",Use specified certificate file to verify the admin server peer."`
```

Comment:
can you make this singular?  just CAcert?

Reply:
Made it CACertFilePath  as per this discussion https://github.com/flyteorg/flyteidl/pull/242#discussion_r773439741

Reply:
Also the flag i felts to keep caCerts since it can also contain a chain of certs to the rootCA  in the same file.
Let me know if you still think we need to make it singular. 
eg : opta-gcp-union-ai-chain.pem
        pta-gcp-union-ai.pem